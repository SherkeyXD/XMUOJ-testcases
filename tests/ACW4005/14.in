8
304889065 126737382
711273084 70228938
65544456 475726545
250133796 38068417
30172552 708421532
103616184 142385129
124822500 284880256
206702710 181303068
