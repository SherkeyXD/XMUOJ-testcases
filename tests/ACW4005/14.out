Link
Link
Joel
Joel
Link
Joel
Joel
Joel
